bus/bus.go

```diff
@@ -967,22 +982,18 @@ func (b *bus) objectEntriesHandlerGET(jc jape.Context, path string) {
 	if jc.DecodeForm("limit", &limit) != nil {
 		return
 	}
-	var prefix string
-	if jc.DecodeForm("prefix", &prefix) != nil {
-		return
-	}
-	bucket := api.DefaultBucketName
-	if jc.DecodeForm("bucket", &bucket) != nil {
+	maxKeys := -1
+	if jc.DecodeForm("maxKeys", &maxKeys) != nil {
```

Comment:
why do we need `maxKeys` and `limit` both? They both mean "don't return more than X objects"

Reply:
Hm I guess I wanted to preserve both `offset+limit` as well as `marker+maxKeys` and return all sorts of errors if the user was mixing and matching but you're right we don't need it at all, removed it 👍